test/hotspot/jtreg/compiler/loopopts/superword/TestMemorySegment.java

```diff
@@ -653,13 +645,9 @@ static Object[] testLongLoop_longIndex_longInvar_sameAdr_byte(MemorySegment a, l
     }
 
     @Test
-    @IR(counts = {IRNode.LOAD_VECTOR_B, "= 0",
-                  IRNode.ADD_VB,        "= 0",
-                  IRNode.STORE_VECTOR,  "= 0"},
-        applyIfPlatform = {"64-bit", "true"},
-        applyIfCPUFeatureOr = {"sse4.1", "true", "asimd", "true"})
     // FAILS: invariants are sorted differently, because of differently inserted Cast.
     // See: JDK-8330274
```

Comment:
8330274 is closed as dup of this one. Should the comment be updated? Same below.

Reply:
I changed 8330274 -> https://bugs.openjdk.org/browse/JDK-8331659 in all the superword tests.